setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
     asgiref
     typing_extensions; python_version < "3.8"
     markupsafe>=1.1.1
+    werkzeug<2.2.0
```

Comment:
Can we add a comment on why we are pinning it to <2.2.0 ?

Reply:
Added the comments